### ISSUE TITLE
请问fcenet是如何达到20fps的

### ISSUE THREAD

**Issue by 6098669** [open-mmlab/mmocr#916]
为啥我测试的只有5.8fps。 如果说减小输入图片尺寸，那不是性能也会下降么。 为啥我测的5.8fps和您之前报道的20fps相差这么大呀
![-5baf53f8fff9dc9f](https://user-images.githubusercontent.com/68067132/162347960-2f89f76d-609e-46a3-a5cc-15381f86e616.png)

**Comment by Mountchicken** [comment]
`5.8 task/s` is not a good reflection of FPS. This is not an average time estimate, but an instantaneous estimate, meaning that the current image inference uses 1/5.8 s. It may just be that the current image inference is slow, and you can observe what this speed is roughly when inferring these 500 images. Or you can count the total inference time and divide it by the number of images to calculate the average FPS

Translated with www.DeepL.com/Translator (free version)

**Comment by xinke-wang** [comment]
Usually, the inference speed FPS only counts the network forward time, but this task/s also includes other data processing time.

**Comment by 6098669** [comment]
感谢各位大佬的回复。另外如图所示，为啥我测试的时候没有eval结果呢，只是个空字典

**Comment by ming-eng** [comment]
请问这个fcenet需要训练多久

**Comment by xinke-wang** [comment]
> 请问这个fcenet需要训练多久

You may check the FCENet training log [here](https://github.com/open-mmlab/mmocr/blob/main/configs/textdet/fcenet/README.md) to estimate the training time.

**Comment by ming-eng** [comment]
当训练fcenet使用默认的ic15配置文件
单卡训练的过程中
损失函数突然为nan
请问您遇到了吗

**Comment by gaotongxiao** [comment]
It happens usually because the learning rate is too large for your batch size. Since the discussion is becoming a bit off-topic, I'm closing this issue now. Please create a new issue for any follow-up discussion.
